sql/catalyst/src/main/scala/org/apache/spark/sql/connector/expressions/expressions.scala

```diff
@@ -115,11 +115,10 @@ private[sql] final case class BucketTransform(
 }
 
 private[sql] object BucketTransform {
```

Comment:
Looks like you should update DataFrameWriter.scala, functions.scala, functions.py, etc related codes. cc @rdblue, @brkyvz and @zero323 FYI